tdvt/tdvt/config_gen/datasource_list.py

```diff
@@ -358,27 +357,55 @@ def load_test(config, test_dir=get_root_dir()):
 class TestRegistry(object):
     """Add a new datasource here and then add it to the appropriate registries below."""
 
-    def __init__(self, ini_file):
+    def __init__(self, ini_file, ds):
         self.dsnames = {}
         self.suite_map = {}
+        self.ds = ds
+
+        # When the ds is a suite, we assign the datasources in the suite to the ds variable.
+        registry_ini_file = get_ini_path_local_first('config/registry', ini_file)
+        self.load_registry(registry_ini_file)
+        self.create_ds_list()
+
+        # Check if any list items are suites themselves. If so we include the datasources in those as well.
+        suites_list = ['all', 'standard', 'all_passing', 'nightly', 'slow', 'all_bigquery_sql', 'all_bigquery', 'all_redshift', 'perf']
```

Comment:
Since the suites are a Tableau-specific thing, we don't want them hard-baked in the source code, will need to figure out how to do this along with how Python figures out what's in a suite.